tomviz/RotateAlignWidget.cxx

```diff
@@ -0,0 +1,401 @@
+  this->Internals->Ui.sliceView->GetRenderWindow()->AddRenderer(
+      this->Internals->mainRenderer.Get());
+  this->Internals->Ui.sliceView_1->GetRenderWindow()->AddRenderer(
+      this->Internals->reconRenderer[0].Get());
+  this->Internals->Ui.sliceView_2->GetRenderWindow()->AddRenderer(
+      this->Internals->reconRenderer[1].Get());
+  this->Internals->Ui.sliceView_3->GetRenderWindow()->AddRenderer(
+      this->Internals->reconRenderer[2].Get());
+
+  vtkNew<vtkInteractorStyleRubberBand2D> interatorStyle;
+  interatorStyle->SetRenderOnMouseMove(true);
+
+  this->Internals->Ui.sliceView->GetRenderWindow()->GetInteractor()->SetInteractorStyle(
+      interatorStyle.Get());
+  this->Internals->Ui.sliceView_1->GetRenderWindow()->GetInteractor()->SetInteractorStyle(
+      interatorStyle.Get());
+  this->Internals->Ui.sliceView_2->GetRenderWindow()->GetInteractor()->SetInteractorStyle(
+      interatorStyle.Get());
+  this->Internals->Ui.sliceView_3->GetRenderWindow()->GetInteractor()->SetInteractorStyle(
+      interatorStyle.Get());
+  this->Internals->setupCameras();
+
+  this->Internals->reconImage->SetOrigin(0, 0, 0);
+  this->Internals->reconImage->SetSpacing(1, 1, 1);
+  this->Internals->reconImage->SetExtent(0, 100, 0, 100, 0, 100);
+  this->Internals->reconImage->AllocateScalars(VTK_FLOAT, 1);
+  this->Internals->reconSliceMapper[0]->SetInputData(this->Internals->reconImage.Get());
+  this->Internals->reconSliceMapper[1]->SetInputData(this->Internals->reconImage.Get());
+  this->Internals->reconSliceMapper[2]->SetInputData(this->Internals->reconImage.Get());
+
+  this->Internals->rotationAxis->SetPoint1(0, 0, 0);
+  this->Internals->rotationAxis->SetPoint1(1, 1, 1);
+  this->Internals->rotationAxis->Update();
+
+  vtkNew<vtkPolyDataMapper> mapper;
+  mapper->SetInputConnection(this->Internals->rotationAxis->GetOutputPort());
+  
+  this->Internals->axisActor->SetMapper(mapper.Get());
+  this->Internals->axisActor->GetProperty()->SetColor(1, 1, 0); // yellow
+  this->Internals->axisActor->GetProperty()->SetLineWidth(2.5);
+  this->Internals->mainRenderer->AddActor(this->Internals->axisActor.Get());
+
+  for (int i = 0; i < 3; ++i)
+  {
+    this->Internals->reconSliceLine[i]->Update();
+    vtkNew<vtkPolyDataMapper> sMapper;
+    sMapper->SetInputConnection(this->Internals->reconSliceLine[i]->GetOutputPort());
+    this->Internals->reconSliceLineActor[i]->SetMapper(sMapper.Get());
+    this->Internals->reconSliceLineActor[i]->GetProperty()->SetColor(1, 0, 0);
+    this->Internals->reconSliceLineActor[i]->GetProperty()->SetLineWidth(2.0);
+    this->Internals->reconSliceLineActor[i]->GetProperty()->SetLineStipplePattern(0xFF00);
+    this->Internals->mainRenderer->AddActor(this->Internals->reconSliceLineActor[i].Get());
+  }
+  this->Internals->Ui.spinBox_1->setValue(25);
+  this->Internals->Ui.spinBox_2->setValue(50);
+  this->Internals->Ui.spinBox_3->setValue(75);
```

Comment:
That's because I was assuming slices from a small (100^3) reconstruction volume was going to be displayed there.  I'll change it when I add the sinograms.